src/pages/Follow/Component/Feed.js

```diff
@@ -0,0 +1,227 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { BsHeart, BsBookmark, BsChat, BsShare } from 'react-icons/bs';
+import { FiMoreHorizontal } from 'react-icons/fi';
+import { useNavigate } from 'react-router';
+
+export const Feed = ({ item }) => {
+  const navigate = useNavigate();
+  const [expand, setExpand] = useState(false);
+
+  const handleExpand = () => {
+    const copy = expand;
+    setExpand(!copy);
```

Comment:
state 자체를 건드리는 게 아니기 때문에 따로 변수 선언하지 않아도 됩니다.
```suggestion
    setExpand(!expand);
```

Reply:
수정하였습니다.